### DISCUSSION TITLE
postgres command necessary?

### DISCUSSION THREAD

**Discussion by titro** [immich-app/immich#10985] [Q&A] [answered]
Hi Folks,

i switched to tensorchord/pgvecto-rs when it was necessary. now i checked the current docker-compose and saw, that a additional postgres command was added to this part:

``#    command: ["postgres", "-c" ,"shared_preload_libraries=vectors.so", "-c", 'search_path="$$user", public, vectors', "-c", "logging_collector=on", "-c", "max_wal_size=2GB", "-c", "shared_buffers=512MB", "-c", "wal_compression=on"]
``

Does somebody know what this is exactly for? Do i have to do additional actions when adding it?
Thanks!

**Comment by mertalev** [comment, marked as answer]
It just does some gentle tuning for performance. It was added to compensate for the overhead of data checksumming.

In your case, the DB won't have checksumming enabled as it can only be enabled when the DB is first created. You can still add that command to benefit from the tuning, though.
